WordPress/src/main/java/org/wordpress/android/ui/layoutpicker/LayoutsRowViewHolder.kt

```diff
@@ -49,7 +51,7 @@ class LayoutsItemViewHolder(
         itemView.updateLayoutParams {
             height = dimensionProvider.rowHeight
         }
-        itemView.findViewById<View>(R.id.layouts_row_separator_line).isVisible = showRowDividers
+        itemView.findViewById<View>(R.id.layouts_row_separator_line).isVisible = isPageLayoutCategory
```

Comment:
I was wondering why I didn't add this initially directly to the `bind` method instead of the lazy block for the `recycler` 🤷 .

Wouldn't it be cleaner there, to keep the scope of the `lazy` block for taking care of the recycler view init?